capybara.gemspec

```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |s|
   s.description = 'Capybara is an integration testing tool for rack based web applications. ' \
                   'It simulates how a user would interact with a website'
 
-  s.files = Dir.glob('{lib,spec}/**/*') + %w[README.md History.md License.txt .yardopts]
+  s.files = Dir.glob('lib/**/*') + %w[README.md History.md License.txt .yardopts]
+
+  # Test files are included in the gem to allow third-party driver packages to
+  # run the Capybara test suite and ensure compatibility.
+  s.test_files = Dir.glob('{lib/capybara/,}spec/**/*')
```

Comment:
The proper way is the original one (add all to `files`). `test_files` had different meaning if I remember well and is not supported anymore (there is no entry at [specification reference](https://guides.rubygems.org/specification-reference/) for example). Some details are at https://github.com/rubygems/rubygems/commit/c07356a2de3576bfc5c11192db99a01037524015.

Reply:
Thanks @simi. TIL.

Updated in [`8fcb5d7`](https://github.com/teamcapybara/capybara/pull/2642/commits/8fcb5d7b927e5e68404a3531a6f754714cb6f21f).